app/models/email_template.rb

```diff
@@ -0,0 +1,18 @@
+class EmailTemplate < ApplicationRecord
+  before_save :ensure_exactly_one_active_per_kind
+
+  enum kind: { signup: 0, trial: 1, assignment: 2 }
+  validates :kind, presence: true
+
+  default_scope { order(created_at: :desc) }
+  scope :active_mail, -> { where(active: true) }
```

Comment:
as @toupeira suggested this also on the `client_notification` I think would make sense to also adapt it here: